Makefile

```diff
@@ -42,6 +43,13 @@ bin/$(ZEEKPATH):
 	@unzip -q bin/$(ZEEKPATH).zip -d bin \
 		&& mv bin/zeek bin/$(ZEEKPATH)
 
+bin/$(SURICATAPATH):
```

Comment:
Would you please `@` the lines below, as the `bin/$(ZEEKPATH)` target does, to remove the download status output?